sdk/core/core-lro/CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # Release History
 
-## 2.1.1 (Unreleased)
+## 2.2.0 (Unreleased)
 
 ### Features Added
 
+- `LroEngine` supports a new `isDone()` function in its options bag which can be used to provide a custom logic for determining when an LRO finished processing.
```

Comment:
Please make sure to add a documentation file explaining how to use this feature, or make an issue to add that later.

Reply:
I will write up a migration guide that describes how to migrate existing lros to use `LroEngine`.